facebook/__init__.py

```diff
@@ -102,6 +102,14 @@ def __init__(self, access_token=None, timeout=None, version=None,
         else:
             self.version = "v" + default_version
 
+    def get_permissions(self, user_id, **args):
```

Comment:
`args` doesn't seem to be used in this method. Is there a reason that it has been included?